Sources/SwiftPlot/Pair.swift

```diff
@@ -11,3 +11,13 @@ public struct Pair<T,U> {
 
 public typealias Point = Pair<Float,Float>
 public let zeroPoint = Point(0.0, 0.0)
+
+public struct Size {
+    public var width: Float
+    public var height: Float
+}
+
+public struct Rect {
+    public var origin: Point
+    public var size: Size
+}
```

Comment:
We could have separate files for these for clarity, or maybe rename this file to Primitives.swift?